### ISSUE TITLE
Export ufuncs from DataArray API

### ISSUE THREAD

**Issue by eric-czech** [pydata/xarray#5261]
Have there been discussions on promoting other ufuncs out of `xr.ufuncs` and into the `DataArray` API like `DataArray.isnull` or `DataArray.notnull`?  

I can see how those two would be an exception given the pandas semantics for them, as opposed to numpy, but I am curious how to recommend best practices for our users as we build a library for genetics on Xarray.

We prefer to avoid anything in our documentation or examples outside of the Xarray API to make things simple for our users, who would likely be easily confused/frustrated by the intricacies of numpy, dask, and xarray API interactions (as we were too not long ago).  To that end, we have a number of methods that produce `NaN` and infinite values, but recommending use of either of these to identify those values via `ds.my_variable.pipe(xr.ufuncs.isfinite)` or `np.isfinite(ds.my_variable)` is not ideal.

I would prefer `ds.my_variable.isfinite()` or maybe even `ds.my_variable.ufuncs.isfinite()`.  Is there a sane way to export all of `xr.ufuncs` from `DataArray`?

**Comment by dcherian** [comment]
I think adding `isfinite` would make sense, since we have `isnull`. 

In general, we are deprecating `xr.ufuncs` in favor of `np.ufunc(DataArray)`

**Comment by eric-czech** [comment]
> In general, we are deprecating xr.ufuncs in favor of np.ufunc(DataArray)

Makes sense.  It is a somewhat awkward distinction to teach though to those who wouldn't appreciate the \_\_array_ufunc__ protocol compliance, especially since most other functionality we appeal to like reductions (e.g. max, min, sum), concat, merge, indexing, filtering, etc. comes through the Xarray APIs alone.

**Comment by seth-p** [comment]
I'm wondering if one could just have a generic implementation of `DataArray.func(*args, **kwargs)` for any unrecognized "`func`" that calls `np.func(self, *args, **kwargs)` (perhaps conditional on some global `numpy_fallthrough` option)?
